lib/beyonic/collection.rb

```diff
@@ -2,4 +2,10 @@
 class Beyonic::Collection < OpenStruct
   include Beyonic::AbstractApi
   set_endpoint_resource "collections"
-end
\ No newline at end of file
+
+  def self.claim(amount, payload={})
```

Comment:
Yes, that's fine and yes, but it can sometimes be null.

Luke.
On Apr 13, 2015 5:38 PM, "Oleg German" <notifications@github.com> wrote:

> In lib/beyonic/collection.rb
> <https://github.com/beyonic/beyonic-ruby/pull/1#discussion_r28281968>:
>
> > \ No newline at end of file
> > +
> > +  def self.claim(amount, payload={})
>
> So I should change it to
>
> self.claim(amount, phonenumber, remote_transaction_id)
>
> ?
>
> BTW staging api seems ignored this parameters, and the only collection
> present at my account (id: 1, organization: 5) has remote transaction id
> equal to null, is it normal?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/beyonic/beyonic-ruby/pull/1/files#r28281968>.
>